src/input-range-slider/input-range-slider.tsx

```diff
@@ -132,8 +136,24 @@ export const InputRangeSlider = ({
         );
     };
 
+    const formatIndicationLabel = () => {
+        return (
+            <LabelText>
+                Searching within&nbsp;
+                {indicationLabelPrefix}
+                {selection}
```

Comment:
this is an array and needs to be formatted for display

ranged values need to be handled as well

e.g.
for single thumb, `search within 2km`
for double thumbs, `search within 2-3km`
for 3 or more thumbs, `search within 2-10km` (if selection contains multiple values, pick the min and max)

Reply:
Updated. thanks for pointing out this